src/NumericalAlgorithms/LinearAlgebra/FindGeneralizedEigenvalues.hpp

```diff
@@ -0,0 +1,51 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines function find_generalized_eigenvalues.
+
+#pragma once
+
+/// \cond
+class DataVector;
+class Matrix;
+namespace gsl {
+template <typename T>
+class not_null;
+} // namespace gsl
+/// \endcond
+
+/*!
+ * \ingroup NumericalAlgorithmsGroup
+ * \brief Solve the generalized eigenvalue problem for two matrices.
+ *
+ * This function uses the lapack routine dggev to solve the
+ * generalized eigenvalue problem \f$A v_a =\lambda_a B v_a \f$
+ * for the generalized eigenvalues \f$lambda_a\f$ and corresponding
+ * eigenvectors \f$v_a\f$.
+ * `A` and `B` are each a `Matrix`; they correspond to square
+ * matrices \f$A\f$ and \f$B\f$ that are the same order \f$N\f$.
```

Comment:
`order` -> `dimension`

Reply:
It looks like you missed this one (I haven't checked the rendered documentation. Will, Larry and I will review later this week)